examples/language_model/data_tools/dataset_utils.py

```diff
@@ -38,6 +37,103 @@
 DSET_TYPES = [DSET_TYPE_BERT, DSET_TYPE_T5, DSET_TYPE_ERNIE]
 
 
+def compile_helper():
+    """Compile helper function ar runtime. Make sure this
+    is invoked on a single process."""
+    import os
+    import subprocess
+    path = os.path.abspath(os.path.dirname(__file__))
+    ret = subprocess.run(['make', '-C', path])
+    if ret.returncode != 0:
+        print("Making C++ dataset helpers module failed, exiting.")
+        import sys
+        sys.exit(1)
+
+
+class BlendableDataset(paddle.io.Dataset):
```

Comment:
Please add more comments, why it's called Blendable

Reply:
The BlendableDataset is a wrapper which used to mix different dataset.
The input is a list of datasets and corresponding weights for each dataset.

`Blendable` means the datasets are combined or mixed together.